### PR TITLE
47332_trainee_show_all_report

### PR DESCRIPTION
**Pull request by asifbuetcse**
reports

### CODE REVIEW

**Comment by Omid07** [comment]
ok

File: app/Http/Controllers/Trainee/ReportsController.php [hunk 1]
```diff
@@ -18,7 +18,8 @@ class ReportsController extends Controller
      */
     public function index()
     {
-        //
+        $reports = \Auth::user()->daily_reports()->get();
```
Comment (dinhhoanglong91) [inline app/Http/Controllers/Trainee/ReportsController.php]: Sorry I forgot commenting this function
https://github.com/framgia/tms_42/blob/develop/app/User.php#L39
It should be `dailyReports`: Function name should be `camelCased`

File: resources/views/trainee/reports/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                        <h2 class="panel-title">
+                            Daily Report for: {!! date_format(new DateTime($report->start_time), 'd-m-Y') !!}
+                        </h2>
+                    </div>
+                    <div class="panel-body">
+                        <table class="table table-bordered">
+                            <tbody>
+                                <tr>
+                                    <td>Start Time</td>
+                                    <td>{!! date("H:i:s",strtotime($report->start_time)) !!}</td>
```
Comment (dinhhoanglong91) [inline resources/views/trainee/reports/index.blade.php]: `date('H:i:s'`
We use `'` for PHP function

**Comment by dinhhoanglong91** [comment]
ok

**Pull request merged**